_documentation/user/flags/colorblindness.md

```diff
@@ -5,4 +5,18 @@ type: Bad
 summary: Can't tell team colors. Don't shoot teammates!
 ---
 
-...
+The Colorblindness flag causes a player to see all players as rogue players and identify no longer identifies players by name for them.
```

Comment:
> "and identify no longer identifies players by name for them."

What?

Reply:
The way it is currently worded may be confusing as we have both an Identify flag that identifies a nearby flag and an identify keybind that identifies a player in view.  So, perhaps something like ".. and identifying players by right clicking or pressing the 'i' key only shows a generic 'looking at player' message."

Reply:
I favor option 3 _unless_ we have "player identify feature" link to somewhere else in the docs describing the feature. Grammatically, there should be parentheses or another comma in option 3.

Reply:
The 'player identify' is usually referred to as an 'aim', as it is used to 'look at' the tank in front, required to trigger GM's follow effect, from where it generally became known as an 'aim'.

Reply:
Notes taken and slight revisions made.